tests/test_utils.py

```diff
@@ -190,3 +190,12 @@ def test_zip_strict():
     # same length, should not raise an error
     for _, _ in zip_strict(list_a, list_b[: len(list_a)]):
         pass
+
+
+def test_cmd_util_rename():
+    """Test that importing cmd_util still works but raises warning"""
+    with pytest.warns(FutureWarning):
+        from stable_baselines3.common import cmd_util
```

Comment:
why not `from stable_baselines3.common.cmd_util import make_vec_env` as this is what people do in practice?

If this fail, it will throw an import error anyway